frontend/src/components/spoilerList/spoiler-list.tsx

```diff
@@ -0,0 +1,116 @@
+import {
```

Comment:
Nitpick: Can you also please use kebab style for `spoilerList`? e.g., `spoiler-list`